### PR TITLE
Add lobster mp workflow

### PR DESCRIPTION
**Pull request by JaGeo**
## Summary

This is an attempt to create an MP-compatible LOBSTER workflow. 

- [x] Tests
- [x] Documentation

### CODE REVIEW

File: src/atomate2/vasp/flows/mp.py [hunk 1]
```diff
@@ -185,3 +190,71 @@ def make(self, structure: Structure, prev_dir: str | Path | None = None) -> Flow
...
+                user_kpoints_settings={"reciprocal_density": 310},
+                user_incar_settings={
+                    "EDIFF": 1e-6,
+                    "NSW": 0,
+                    "LWAVE": True,
+                    "ISYM": 0,
+                    "IBRION": -1,
+                    "ISMEAR": -5,
+                    "LORBIT": 11,
+                    # "ICHARG": 0, # is this okay?
```
Comment (JaGeo) [inline src/atomate2/vasp/flows/mp.py:251]: It would be good to test both runs with and without structure optimization. There could be differences.

Comment (naik-aakash) [reply src/atomate2/vasp/flows/mp.py:251]: Sure will test these :smiley:

**Comment by JaGeo** [comment]
@utf , @naik-aakash has now finished this PR. It is a LOBSTER workflow that is compatible with our LOBSTER database. This might also be useful for future integration of data into the MP. We would be happy about another review.

**Review by utf — COMMENTED** [review]
Thanks @JaGeo and @naik-aakash. All looks good to me.

One question I had that could maybe be clarified in the docs: what do you mean by MP compatible settings - I see you're changing the pseudo potential version to PBE_54?

**Comment by JaGeo** [comment]
> Thanks @JaGeo and @naik-aakash. All looks good to me.
> 
> One question I had that could maybe be clarified in the docs: what do you mean by MP compatible settings - I see you're changing the pseudo potential version to PBE_54?

Yes, you are right. We should maybe clarify that it is compatible with our Lobster database that will go into MP at some point.

File: docs/user/codes/vasp.md [hunk 2]
```diff
@@ -258,6 +258,13 @@ VASP_CMD: <<VASP_CMD>>
 LOBSTER_CMD: <<LOBSTER_CMD>>
 ```
 
+```{note}
+Materials project compatible settings LOBSTER workflow is also available now,
```
Comment (JaGeo) [inline docs/user/codes/vasp.md:262]:
```suggestion
Settings compatible to LOBSTER database (Naik, A.A., Ertural, C., Dhamrait, N. et al. A Quantum-Chemical Bonding Database for Solid-State Materials. Sci Data 10, 610 (2023). https://doi.org/10.1038/s41597-023-02477-5 , currently being integrated into Materials Project)LOBSTER workflow is also available now,
```

Comment (naik-aakash) [reply docs/user/codes/vasp.md]: Thanks 😄

**Comment by utf** [comment]
Thanks!

**Pull request merged**